doc/tutorial.rst

```diff
@@ -89,117 +97,140 @@ We will use the following very simple Dockerfile:
 This creates a minimal CentOS 8 image with :code:`ssh` installed. We
 will encounter more complex Dockerfiles later in this tutorial.
 
-.. note::
-
-   Docker does not update the base image unless asked to. Specific images can
-   be updated manually; in this case::
+Build Charliecloud image
+------------------------
 
-     $ sudo docker pull centos:8
+The three arguments here are the :code:`ch-image` subcommand :code:`build`,
+the option to enable unprivileged build workarounds :code:`--force`, and the
```

Comment:
I'm nervous about including `--force` without explanation, but it's probably good enough for now.